com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -370,12 +370,12 @@ private static ArrayList<File> getExtenderAssets(Project project) throws IOExcep
     * Get a list of dex files to include in the aab
     */
     private static ArrayList<File> getClassesDex(Project project) throws IOException {
-        ArrayList<File> classesDex = new ArrayList<File>();
+        ArrayList<File> classesDex = new ArrayList<>();
+        boolean hasExtensions = ExtenderUtil.hasNativeExtensions(project);
```

Comment:
Use here `ExtenderUtil.hasNativeExtensions` to define is binary vanilla engine or not. Because of we donwload unstripped version in `Project.java` we place that binary in build folder. Old logic depends on is that binary exists or not. If it exists - `AndroidBundler` decided that engine arrived from Extender and tried to find classes.dex in the build result. So now we determine if it vanilla build or not to decide we to locate clasess.dex.